play/src/containers/FinalResultsContainer.tsx

```diff
@@ -46,7 +61,11 @@ export default function FinalResultsContainer({
         <Congrats
           score={score}
           selectedAvatar={selectedAvatar}
-          leader={leader}
+          leader={
+            !isNullOrUndefined(teams) && !isNullOrUndefined(teamId)
+              ? isLeader(teams, teamId)
```

Comment:
we're going to be calling isLeader every time this renders. Can we use useRef like Josephina did in this [PR](https://github.com/rightoneducation/righton-app/pull/683/files) to prevent this from happening? we might want to move the `!isNullOrUndefined(teams) && !isNullOrUndefined(teamId)` into the function itself as well.